introduction.md

```diff
@@ -1 +1,3 @@
 Hey! This is me. I looove JavaScript.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?